src/fluka/fluka_funcs.cpp

```diff
@@ -954,7 +955,7 @@ void fludagwrite_assignma(std::ostringstream& ostr,
       // otherwise we assume uwuw
       // if you arent graveyard or vacuum you must be in the uwuw library
       if (mat_prop.find("Graveyard") == std::string::npos && mat_prop.find("Vacuum") == std::string::npos) {
-        pyne::Material material = pyne_map[mat_prop];
+        pyne::Material material = *(pyne_map.at(mat_prop));
```

Comment:
if you keep the pyne::MaterialLibrary you could use get_material method and or the [] operator